Putting a manually typed version in pyproject.toml, as it wouldnt resolve automatically.
Author: Denis Kristak (Inuits)
diff -ruN cppy-1.2.1_orig/pyproject.toml cppy-1.2.1/pyproject.toml
--- cppy-1.2.1_orig/pyproject.toml	2022-03-30 21:39:41.000000000 +0200
+++ cppy-1.2.1/pyproject.toml	2023-08-14 14:04:36.664029166 +0200
@@ -8,6 +8,7 @@
 
 [project]
 name = "cppy"
+version = "1.2.1"
 readme = "README.rst"
 requires-python = ">=3.7"
 license = {file = "LICENSE"}
@@ -27,8 +28,7 @@
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: Implementation :: CPython",
 ]
-dynamic=["version"]
-
+dynamic=[]
 
 [project.urls]
 homepage = "https://github.com/nucleic/cppy"
@@ -36,42 +36,10 @@
 repository = "https://github.com/nucleic/cppy"
 changelog = "https://github.com/nucleic/cppy/blob/main/releasenotes.rst"
 
-
 [build-system]
-requires = ["setuptools>=61.2", "wheel", "setuptools_scm[toml]>=3.4.3"]
+requires = ["setuptools>=61.2", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
 packages = ["cppy"]
-package-data = {cppy = ["include/cppy/*.h"]}
-
-[tool.setuptools_scm]
-write_to = "cppy/version.py"
-write_to_template = """
-# -----------------------------------------------------------------------------
-# Copyright (c) 2014-2022, Nucleic Development Team.
-#
-# Distributed under the terms of the Modified BSD License.
-#
-# The full license is in the file LICENSE, distributed with this software.
-# -----------------------------------------------------------------------------
-# This file is auto-generated by setuptools-scm do NOT edit it.
-
-from collections import namedtuple
-
-#: A namedtuple of the version info for the current release.
-_version_info = namedtuple("_version_info", "major minor micro status")
-
-parts = "{version}".split(".", 3)
-__version_info__ = _version_info(
-    int(parts[0]),
-    int(parts[1]),
-    int(parts[2]),
-    parts[3] if len(parts) == 4 else "",
-)
-
-# Remove everything but the 'version_info' from this module.
-del namedtuple, _version_info, parts
-
-__version__ = "{version}"
-"""
+package-data = {cppy = ["include/cppy/*.h"]}
\ No newline at end of file